interfaces.go

```diff
@@ -143,8 +146,42 @@ type Exception struct {
 
 type EventID string
 
-// https://docs.sentry.io/development/sdk-dev/event-payloads/
+// TraceContext describes the context of the trace.
```

Comment:
```suggestion
// TraceContext describes the context of the trace.
//
// Experimental: This is part of a beta feature.
```

The extra line creates a new paragraph, may look neater in rendered form.